experimental/segmenter/src/line.rs

```diff
@@ -99,7 +99,10 @@ impl Default for LineBreakOptions {
 pub struct LineBreakSegmenter {
     options: LineBreakOptions,
     payload: DataPayload<LineBreakDataV1Marker>,
-    dictionary_payload: DataPayload<UCharDictionaryBreakDataV1Marker>,
+    dictionary_km_payload: Option<DataPayload<UCharDictionaryBreakDataV1Marker>>,
+    dictionary_lo_payload: Option<DataPayload<UCharDictionaryBreakDataV1Marker>>,
```

Comment:
Question: No CJK dict here?

Reply:
CJ dictionary is unnecessary for line segmenter since UAX#14 has CJ rules.